timeless/restaurants/floors/views.py

```diff
@@ -55,12 +53,11 @@ def edit(id):
         )
 
 
-@BP.route("/delete", methods=["POST"])
-@auth.login_required
-def delete():
-    """ Delete floor with id """
-    flash("Delete not yet implemented")
-    return redirect(url_for("floor.list_floors"))
+class Delete(views.DeleteView):
+    """ Delete floor with id """  
+    decorators = (auth.login_required,)
```

Comment:
@emilianodellacasa Can you remove the ``,``?

Reply:
@amihaiemil No, it's a list